.github/workflows/beam_Python_ValidatesContainer_Dataflow_ARM.yml

```diff
@@ -122,4 +122,4 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: python-code-coverage-report
-          path: "**/pytest*.xml"
\ No newline at end of file
+          path: "**/pytest*.xml"
```

Comment:
Maybe remove the white space? Not sure why there is always white space being added every time a yml file is edited, though didn't tough the last line :(

Reply:
This is a consequence of using the GitHub editor (as opposed to local), I think it just changes some of the line endings. I'm inclined to just leave it and ignore any of these changes

Reply:
SGTM, thanks!